src/schemas/space.json

```diff
@@ -326,6 +326,39 @@
             "additionalProperties": false
           }
         },
+        "nftClaimer": {
+          "type": "object",
+          "properties": {
+            "maxSupply": {
+              "type": "number",
+              "minimum": 0
+            },
+            "mintPrice": {
+              "type": "number",
+              "minimum": 0
+            },
+            "proposerCut": {
+              "type": "number",
+              "minimum": 0
+            },
+            "address": {
+              "type": "string",
+              "title": "Contract address",
+              "examples": ["e.g. 0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984"],
+              "pattern": "^0x[a-fA-F0-9]{40}$",
+              "minLength": 42,
+              "maxLength": 42
+            },
+            "network": {
+              "type": "string",
+              "title": "Network",
+              "maxLength": 12
+            },
+            "enabled": {
+              "type": "boolean"
+            }
```

Comment:
I am not following much but do we need `enabled`? if we have `nftClaimer` other than object, can we consider it as enabled? or do we want to keep the settings when it is disabled?

Reply:
NFT Claimer can be enabled, then disabled later

Reply:
I mean if it is disabled, do we want to keep the previous settings?

Reply:
yes, if because he can enable it back

Reply:
And there may be already minted nft

Reply:
I'd suggest you add Title, Description and Examples to each field, then you will be able to use `TuneForm` to easily generate the form on the Ui. Example here: https://tune-histoire.vercel.app/ see `Source` in the sidebar for the used Schema

Reply:
Added